build.sh

```diff
@@ -24,4 +24,4 @@ cmake -DCMAKE_BUILD_TYPE=Release \
   -DDEPOT_TOOLS_DIR="$root/depot_tools" \
   "$@"
 
-(cd $root/build && ninja && ctest --output-on-failure -R _rpunit)
+(cd $root/build && ninja && LD_LIBRARY_PATH=$PWD/deps_install/lib ctest --output-on-failure -R _rpunit)
```

Comment:
yeah, i'll double check that it's needed. i recall it was required when i was working with this earlier but then a bunch of changes went in and didn't check if removing it would still make it